bake.hcl

```diff
@@ -0,0 +1,116 @@
+variable "IMAGE_TAG" {
+  default = "4.x"
+}
+
+variable "DOCKERHUB_NAMESPACE" {
+  default = "singledigital"
+}
+
+variable "CONTEXT" {
+  default = "bay/images"
+}
+
+variable "LAGOON_IMAGE_VERSION" {
+  # Used to control which version of upstream Lagoon
+  # images the bay images are built from.
+  default = "latest"
+}
+
+group "default" {
+    targets = [
+      "ci-builder",
+      "mariadb",
+      "cli",
+      "php",
+      "nginx-php",
+      "node",
+      "circle",
+    ]
+}
+
+target "ci-builder" {
+  context       = "${CONTEXT}"
+  dockerfile    = "Dockerfile.ci-builder"
```

Comment:
Unlike compose - `buildx bake` treats the context as a global location - if you reference the image files it will try to include them from a nested path.

```
dockerfile = "bay/images/Dockerfile.ci-builder"
```

Will try and include:

```
bay/images/bay/images/Dockerfile.ci-builder
```